index.html

```diff
@@ -119,6 +126,35 @@ <h4>LCHS</h4>
                         </div><!--end card-->
                     </a>
                 </div><!-- col sm 6-->
+              
+                 <div class="col-sm-6 col-md-4">
+                    <a class="learn-card" href="https://launchcodetechnicaltraining.org/linux/">
+                        <div class="card">
+                            <div class="card-header">
+                                <h4>Linux</h4>
+                            </div>
+                            <div class="card-content">
+                                <p><span class="card-inline-title">Linux</span>This course is an introduction to the Linux and the Command Line.
```

Comment:
```suggestion
                                <p><span class="card-inline-title">Linux</span> course introduces the fundamentals of working within a Linux Operating System.  Topics include deploying web applications, working with servers, and using the Command Line.
```

Reply:
Was trying to get the Linux link to show up.  Shortened the course summary, too.